packages/core/src/components/Table/stories/TableSamples/LoadingContainer.tsx

```diff
@@ -10,50 +8,32 @@ import { hexToRgbA } from "@core/utils/hexToRgbA";
 import { HvLoading } from "@core/components/Loading";
 import { useTheme } from "@core/hooks/useTheme";
 
-const styles = {
-  loading: css({
-    width: "100%",
-    height: "100%",
-  }),
-  overlay: css({
-    position: "absolute",
-    transition: "background-Color .2s ease",
-    zIndex: -1,
-  }),
-  blur: css({
-    backgroundColor: hexToRgbA("#EEEEEE"),
-    zIndex: theme.zIndices.popover,
-  }),
-};
-
 const LoadingContainer = ({ children, loading, ...others }) => {
-  const ref = useRef(null);
-  const [overlayPosition, setOverlayPosition] = useState({});
   const { colors } = useTheme();
 
-  useEffect(() => {
-    if (children && ref.current) {
-      const { clientHeight, clientWidth, offsetTop, offsetLeft } = ref.current;
-      setOverlayPosition({
-        top: offsetTop,
-        left: offsetLeft,
-        height: clientHeight,
-        width: clientWidth,
-      });
-    }
-  }, [children]);
+  const { ref } = useResizeDetector();
```

Comment:
Probably much better than the `useEffect`, but JS for a simple container overlay seems overkill - this can easily be done with just CSS

Reply:
Yup, I was actually updating this at the moment to remove this `useResizeDetector`